config/initializers/0_ldap_authenticatable.rb

```diff
@@ -0,0 +1,109 @@
+require 'net/ldap'
```

Comment:
Yep, but how can we do otherwise?
As far as I know (little experience with other projects), config/initializer files are supposed to be ordered, it's just luck that makes the ones in mastodon not necessarily ordered?